### ISSUE TITLE
Add a table of all transactions

### ISSUE THREAD

**Issue by DanThrane** [SDU-eScience/UCloud#1986]

**Comment by Tiggles** [comment]
Any update on this? I think this needed some backend work.

**Comment by DanThrane** [comment]
It does. This is currently blocked by #2169. I think it was supposed to be done with the comment issue but it does not appear to have been implemented.

**Comment by DanThrane** [comment]
Low priority. Let's reevaluate later.
